khmer/__init__.py

```diff
@@ -287,4 +287,27 @@ class HLLCounter(_HLLCounter):
     """
 
     def __len__(self):
-        return self.estimate_cardinality()
+        """Return the current estimation."""
+        return self._estimate_cardinality()
+
```

Comment:
Please don't rename the CPython methods. Just call the method directly `return self.estimate_cardinality()`